src/main/java/com/google/firebase/remoteconfig/TagColor.java

```diff
@@ -0,0 +1,26 @@
+package com.google.firebase.remoteconfig;
+
+public enum TagColor {
+  BLUE("BLUE"),
+  BROWN("BROWN"),
+  CYAN("CYAN"),
+  DEEP_ORANGE("DEEP_ORANGE"),
+  GREEN("GREEN"),
+  INDIGO("INDIGO"),
+  LIME("LIME"),
+  ORANGE("ORANGE"),
+  PINK("PINK"),
+  PURPLE("PURPLE"),
+  TEAL("TEAL"),
+  UNSPECIFIED("CONDITION_DISPLAY_COLOR_UNSPECIFIED");
```

Comment:
Just `UNSPECIFIED`?

Reply:
The rest API is looking for the string `CONDITION_DISPLAY_COLOR_UNSPECIFIED` or `null`
https://firebase.google.com/docs/reference/remote-config/rest/v1/RemoteConfig#conditiondisplaycolor